apps/web/public/static/locales/en/common.json

```
@@ -1511,6 +1511,7 @@
   "no_available_apps": "There are no available apps",
   "no_available_apps_description": "Please ensure there are apps in your deployment under 'packages/app-store'",
   "no_apps": "There are no apps enabled in this instance of Cal",
+  "no_apps_contact_admin": "No app has been configured yet. Please contact your admin",
```

```suggestion
  "no_apps_contact_admin": "No app has been configured yet",
```


lets add "Please contact your admin" to the description instead